[DOC-1322] Docs for Voyager v2026.5.2 (#31891)
yugabyte/yugabyte-db: docs/content/stable/yugabyte-voyager/migrate/live-fall-back.md

Perform the following steps as part of the cutover process:

> 1. Quiesce your source database, that is stop application writes.

Reviewer: ```suggestion
1. Stop application writes on the source database.
```

   END LOOP;
END $$;

-- SCHEMA_LIST is a comma-separated list of schemas, for example: 'abc', 'public', 'xyz'
```

{{% /tab %}}

{{% tab header="Oracle" %}}

Disable triggers and foreign-key constraints on the source database to ensure that changes from the target YugabyteDB database can be imported correctly to the source database using the following PL/SQL commands on the source schema as a privileged user:
Use the following PL/SQL commands to disable triggers, and disable referential constraints on the source:

```sql
-- Disable triggers
BEGIN
   FOR R IN (SELECT owner, object_name FROM all_objects WHERE owner=UPPER('<SCHEMA_NAME>') and object_type ='TABLE' MINUS SELECT owner, table_name from all_nested_tables where owner = UPPER('<SCHEMA_NAME>'))
   LOOP
     EXECUTE IMMEDIATE 'ALTER TABLE '||R.owner||'."'||R.object_name||'" DISABLE ALL TRIGGERS';
   END LOOP;
END;
/

-- Disable referential constraints
BEGIN
   FOR c IN (SELECT table_name, constraint_name
     FROM user_constraints
     WHERE constraint_type IN ('R') AND OWNER = '<SCHEMA_NAME>')
   LOOP
     EXECUTE IMMEDIATE 'ALTER TABLE ' || c.table_name || ' DISABLE CONSTRAINT ' || c.constraint_name;
   END LOOP;
END;

> /

Author: @priyanshi-yb - not sure but tagging you about these gemini suggestions.

> /

Reviewer 2: yeah, lets not change anything on these PLPGSQL scripts, right now. I will take these and see if we need to do anything later
